stubs/click-default-group/METADATA.toml

```
@@ -0,0 +1,5 @@
+version = "1.2.*"
+requires = ["click"]
+
+[tool.stubtest]
+ignore_missing_stub = false
```

This is now the default and is no longer needed

```suggestion
[tool.stubtest]
ignore_missing_stub = false
```